resources/VirtualMachine.py

```diff
@@ -0,0 +1,6 @@
+class VirtualMachine:
```

Comment:
```suggestion
class VirtualMachine(Host):
```

Adding the parent relation like this will be helpful to go up the tree later. Please do it for the other classes to up to Vcenter